diff -urN barrage-1.0.4/barrage.desktop barrage-1.0.4-patched/barrage.desktop
--- barrage-1.0.4/barrage.desktop	2011-07-29 02:00:11.000000000 +1100
+++ barrage-1.0.4-patched/barrage.desktop	2011-12-14 18:23:50.694362332 +1100
@@ -1,7 +1,8 @@
 [Desktop Entry]
 Name=Barrage
-Comment=A rather destructive action game.
+Comment=A rather destructive action game
 Exec=barrage
-Icon=barrage.png
+Icon=barrage
 Type=Application
-Categories=Game;
+Categories=Game;ArcadeGame;
+Terminal=false
